megalodon/src/entities/status.ts

```diff
@@ -37,7 +37,7 @@ namespace Entity {
     application: Application | null
     language: string | null
     pinned: boolean | null
-    emoji_reactions: Array<Reaction>
+    reactions: Array<Reaction>
```

Comment:
Why did you change the name?

Reply:
More descriptive, given reactions can be unicode emojis, custom emojis, or technically even arbitrary text or images

Reply:
If you want, I can undo that to keep it as `emoji_reactions`

Reply:
No problem. However, since it is a breaking change, I'm wondering when to merge it.

Reply:
Hopefully sooner than later, I'd love to use upstream Megalodon inside Firefish :)

Reply:
Can you please move this change to another PR?
The field renaming should not be the main target of this PR. And if you separate it, I'll include it in the Firefish support release.